common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
With those lines of code I get weird logs:
```
WAIT FOR STATE: playing
Current state of '.video' element is 'is-buffered', position is '2:10'
Current state of '.video' element is 'is-buffered', position is '2:10'
Current state of '.video' element is 'is-playing', position is '2:10'
...
WAIT FOR STATE: pause
Current state of '.video' element is 'is-playing', position is '2:10'
...
Current state of '.video' element is 'is-playing', position is '2:11'
...
Current state of '.video' element is 'is-playing', position is '2:12'
...
Current state of '.video' element is 'is-playing', position is '2:13'
...
Current state of '.video' element is 'is-playing', position is '2:14'
Current state of '.video' element is 'is-paused', position is '2:15'
WAIT FOR STATE: playing
Current state of '.video' element is 'is-buffered', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-paused', position is '2:15'

------------------------------------------------------------------------------------------
Current state of '.video' element is 'is-paused', position is '2:15'
Current state of '.video' element is 'is-ended', position is '2:20'
------------------------------------------------------------------------------------------
``` 
It looks like the Promise doesn't work between `2:15` and `2:20` video positions.
[BDD](https://github.com/edx/edx-platform/blob/anton/fix-flaky-tests/common/test/acceptance/tests/video/test_video_times.py#L107).
@jzoldak Any ideas?
When I comment these lines of code, the tests look stable.
https://build.testeng.edx.org/job/edx-platform-all-tests-flow/1241/
https://build.testeng.edx.org/job/edx-platform-all-tests-flow/1248/
https://build.testeng.edx.org/job/edx-platform-all-tests-flow/1264/
https://build.testeng.edx.org/job/edx-platform-all-tests-flow/1327/
https://build.testeng.edx.org/job/edx-platform-all-tests-flow/1332/